build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -25,29 +21,31 @@
   <Target Name="Build"
           DependsOnTargets="LogOutput;
                             InvalidateNuGetCache;
-                            @(NuSpec->'%(LayoutTargetName)')"
-          Inputs="@(NuSpec);@(PackageAssets)"
-          Outputs="@(NuSpec->'%(NuPkgOutput)')">
+                            @(NuSpec->'%(LayoutTargetName)')">
 
-    <PropertyGroup>
-      <NuGetExeVersion>3.4.4</NuGetExeVersion>
-      <NuGetTool>$(RepositoryRootDirectory).nuget\$(NuGetExeVersion)\nuget.exe</NuGetTool>
-      <NuGetDownloadUrl>https://dist.nuget.org/win-x86-commandline/v$(NuGetExeVersion)/NuGet.exe</NuGetDownloadUrl>
-      <NuGetDownloadCommand>powershell -noprofile -nologo -command "(new-object System.Net.WebClient).DownloadFile('$(NugetDownloadURL)', '$(NuGetTool)')"</NuGetDownloadCommand>
-    </PropertyGroup>
+    <ItemGroup>
+      <BinPlaceContent Include="$(MSBuildThisFileDirectory)Sdkproject.csproj" />
+      <BinPlaceContent Include="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.nuspec" />
+      <NuspecFile Include="$(InitialOutDir)Microsoft.NET.Sdk.nuspec" />
+    </ItemGroup>
 
+    <Copy 
+        SourceFiles="@(BinPlaceContent)"
+        DestinationFolder="$(InitialOutDir)"
+    />
+    <Exec Command="$(DotNetTool) restore $(InitialOutDir)" />
     <MakeDir Directories="$(NuGetOutDir)" />
+    <ReadLinesFromFile File="@(NuspecFile)" >
+         <Output TaskParameter="Lines" ItemName="ItemsFromFile" />
+    </ReadLinesFromFile>
+    <ItemGroup>
+        <NewLines Include="@(ItemsFromFile -> '$([System.String]::Copy('%(ItemsFromFile.Identity)').Replace('$version$','$(Version)'))' )" />
```

Comment:
Does the `dotnet pack` command not have a way to specify the version to use when packing a .nuspec?  If not, can we file a bug to get that added (nuget.exe supports this).